onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -827,8 +828,10 @@ ::std::vector<::std::basic_string<ORTCHAR_T>> GetParameterStrings() {
           v.emplace_back(r);
           return true;
         });
-      } catch (std::exception&) {
+      }
+      ORT_CATCH(std::exception&) {
       }  // ignore non-exist dir
```

Comment:
nit: would be good to find-and-replace 'ORT_CATCH(std::exception' with 'ORT_CATCH(const std::exception'